web/src/lib/domain-logic/shouldShowDisclaimerForProfileNotSubmittingData.ts

```diff
@@ -0,0 +1,24 @@
+import { IsAuthenticated } from "@/lib/auth/AuthContext";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+
+const isFormedOutsideNavigator = (business: Business): boolean => {
```

Comment:
So I thought about making this its own function in its own file with its own tests. But I think that might a refactor if we ever need it somewhere else rather than something to do pre-emptively.

Reply:
I don't think it necessary to extract it out, if we ever do, at that point it would make sense to create test. I see it as a helper function that only lives in this file